spec/controllers/group_assignment_invitations_controller_spec.rb

```diff
@@ -58,6 +58,24 @@
         expect(group_assignment.group_assignment_repos.count).to eql(0)
         expect(user.repo_accesses.count).to eql(0)
       end
+
+      it 'does not allow user to join if group has reached maximum number of members' do
```

Comment:
You should also add a `context 'group has not reached maximum number of members'` with tests so that you have all permutations covered.